lib/segment/src/index/field_index/geo_index/immutable_geo_index.rs

```diff
@@ -53,6 +53,18 @@ impl ImmutableGeoMapIndex {
         &self.db_wrapper
     }
 
+    pub fn get_points_count(&self) -> usize {
+        self.points_count
+    }
+
+    pub fn get_points_values_count(&self) -> usize {
+        self.points_values_count
+    }
+
+    pub fn get_max_values_per_point(&self) -> usize {
+        self.max_values_per_point
+    }
```

Comment:
Naming nit: don't use `get_` prefix. https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

```suggestion
    pub fn points_count(&self) -> usize {
        self.points_count
    }

    pub fn points_values_count(&self) -> usize {
        self.points_values_count
    }

    pub fn max_values_per_point(&self) -> usize {
        self.max_values_per_point
    }
```

Reply:
Fixed for all functions in geo index